components/PicturesListComponent.js

```diff
@@ -1,54 +1,41 @@
 import React from 'react';
-import {FlatList, ActivityIndicator, StyleSheet, View} from 'react-native';
+import {FlatList, ActivityIndicator, StyleSheet, View, TouchableOpacity, Text} from 'react-native';
 import PictureItemComponent from './PictureItemComponent';
-import {Button} from 'react-native-elements';
 
 const PicturesListComponent = props => {
-  const {isLoading, data, fetchData, sortDataByAuthor, sortDataById} = props;
-
-  const renderItem = ({item}) => <PictureItemComponent item={item} />;
+  const {isLoading, data, fetchData, sortDataByAuthor, sortDataById, loadInBrowser} = props;
 
   return (
     <>
-      {isLoading && (
-        <View style={styles.indicatorContainer}>
-          <ActivityIndicator size="large" color="dodgerblue" animating />
-        </View>
-      )}
-
-      {!isLoading && (
-        <FlatList
+      {isLoading === true
```

Comment:
isLoading - should be boolean so you don't need to compare it - it should be a condition state itself